src/System.Private.ServiceModel/tests/Scenarios/Client/ChannelLayer/RequestReplyChannelShapeTests.cs

```diff
@@ -121,4 +122,114 @@ public static void IRequestChannel_Http_CustomBinding()
             ScenarioTestHelpers.CloseCommunicationObjects(channel, factory);
         }
     }
+
+    [Fact]
+    [OuterLoop]
+    public static void IRequestChannel_Async_Http_BasicHttpBinding()
+    {
+        IChannelFactory<IRequestChannel> factory = null;
+        IRequestChannel channel = null;
+        Message replyMessage = null;
+
+        try
+        {
+            // *** SETUP *** \\
+            BasicHttpBinding binding = new BasicHttpBinding(BasicHttpSecurityMode.None);
+
+            // Create the channel factory
+            factory = binding.BuildChannelFactory<IRequestChannel>(new BindingParameterCollection());
```

Comment:
Nit: not required to change.  We probably should make the async channel factory tests separate so that the EXECUTE section involves only the async channel factory open.  I like to have tests where the SETUP part uses only code that is covered in some other test's EXECUTE.  That way it is clear what is being tested in a test, and what is assumed to work.  Don't feel obliged to change this PR for that.